pylxd/image.py

```diff
@@ -84,3 +84,33 @@ def export(self):
         """Export the image."""
         response = self.api.export.get()
         return response.content
+
+    def add_alias(self, name, description):
+        """Add an alias to the image."""
+        self.client.api.images.aliases.post(json={
+            'description': description,
+            'target': self.fingerprint,
+            'name': name
+        })
+
+        # Update current aliases list
+        self.aliases.append({
+            'description': description,
+            'target': self.fingerprint,
+            'name': name
+        })
+
+        return self
+
+    def delete_alias(self, name):
+        """Delete an alias from the image."""
+        self.client.api.images.aliases[name].delete()
+
+        # Update current aliases list
+        la = [a['name'] for a in self.aliases]
+        try:
+            del self.aliases[la.index(name)]
+        except ValueError:
```

Comment:
You're missing code coverage here. I wonder why it is that we would need to add the `ValueError` here. When would we hit a case where the alias doesn't exist? If there are valid cases, we should list them in a comment here.